packages/components/nodes/embeddings/GoogleVertexAIEmbedding/GoogleVertexAIEmbedding.ts

```diff
@@ -0,0 +1,57 @@
+import { GoogleVertexAIEmbeddings, GoogleVertexAIEmbeddingsParams } from 'langchain/embeddings/googlevertexai'
+import { INode, INodeData, INodeParams } from '../../../src/Interface'
+import { getBaseClasses } from '../../../src/utils'
+
+class GoogleVertexAIEmbedding_Embeddings implements INode {
+    label: string
+    name: string
+    version: number
+    type: string
+    icon: string
+    category: string
+    description: string
+    baseClasses: string[]
+    credential: INodeParams
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'GoogleVertexAI Embeddings'
+        this.name = 'googlevertexaiEmbeddings'
+        this.version = 1.0
+        this.type = 'GoogleVertexAIEmbeddings'
+        this.icon = 'vertexai.svg'
+        this.category = 'Embeddings'
+        this.description = 'Google vertexAI API to generate embeddings for a given text'
+        this.baseClasses = [this.type, ...getBaseClasses(GoogleVertexAIEmbeddings)]
+        this.inputs = [
+            {
+                label: 'Model Name',
+                name: 'modelName',
+                type: 'options',
+                options: [
+                    {
+                        label: 'textembedding-gecko',
+                        name: 'textembedding-gecko'
+                    },
+                    {
+                        label: 'textembedding-gecko@001',
```

Comment:
I guess this model name might be fix version of model like openAI do.
currently, I am contacting with google and after reply coming, I will write content of their reply

Reply:
I asked google and thire answer is

`@001` is for version settings